pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
I have attached a file to my previous message, with a code that reproduces the error.
Please let me know if you can't reproduce it.
The file was created in google slides (error still there if you open and save it in powerpoint).
Error is present if you have at least two images in template, so when python runs sorted([None, None]).